legal-api/src/legal_api/services/authz.py

```diff
@@ -533,6 +543,23 @@ def has_blocker_invalid_state_filing(state_filing: Filing, blocker_checks: dict)
     return has_state_filing(state_filing, state_filing_types)
 
 
+def has_blocker_completed_filing(completed_filing: Filing, blocker_checks: dict):
+    """Check if business has an completed filing."""
+    if not (complete_filing_types := blocker_checks.get('completedFilings', [])):
+        return False
+
+    if not completed_filing:
+        return False
+
+    if not has_state_filing(completed_filing, complete_filing_types):
+        return False
+
+    if completed_filing.status == Filing.Status.COMPLETED.value:
+        return True
+
+    return False
```

Comment:
The code here should be a check to see if the filing types provided(if any) in the `completedFilings` list exist for the business and have a filing status of `COMPLETED`.   If not return `False`.